### ISSUE TITLE
Show Timezone on Restore Time check

### ISSUE THREAD

**Issue by My1** [digitalbitbox/bitbox02-firmware#580]
the current time (converted to a UTC Unixtime) will be used to set the U2F counter in a way that even on a backup the counter wouldnt interfere (provided 2 devices arent used at the same time), however this could get weird when the displayed "local" date/time on the PC matches but the timezone doesnt and a U2F counter could be influenced solely by timezone of around 93,6k by the pure time difference of the timezones (not sure how this goes when you put the date line into the calculation)

if the relying party (website) uses the counter very strictly, this could lead to unintended or malicious lockouts, which would be very annoying.

**Comment by benma** [comment]
Can you rephrase this / elaborate? The timezone info is only to display the time correctly to the user and is not used for the U2F counter. 

Also a website using U2F can always lock you out if they want to.
